### PR TITLE
 Moved Declaration of dynamic content from Makefile to .env file for Equinix-Metal

### PR DESCRIPTION
**Pull request by piyuagr**
**What this PR does / why we need it**:
This PR updates Makefile content to import dynamic content from the .env file.

**Which issue(s) this PR fixes**:
Fixes -> https://github.com/gardener/machine-controller-manager/issues/846

**Special notes for your reviewer**:

**Release note**:

```feature user
NONE
```

### CODE REVIEW

**Comment by ghost** [comment]
@piyuagr Thank you for your contribution.

**Review by schrodit — APPROVED** [review]
/lgtm

File: Makefile [hunk 1]
```diff
@@ -18,9 +18,8 @@ IMAGE_REPOSITORY    := <link-to-image-repo>
 IMAGE_TAG           := $(shell cat VERSION)
 PROVIDER_NAME       := SampleProvider
 PROJECT_NAME        := gardener
-CONTROL_NAMESPACE  := default
-CONTROL_KUBECONFIG := dev/target-kubeconfig.yaml
-TARGET_KUBECONFIG  := dev/target-kubeconfig.yaml
+
+-include .env
```
Comment (himanshu-kun) [inline Makefile]: include directives should be the first lines in the file, after the copyright.. kindly update

**Review by himanshu-kun — CHANGES_REQUESTED** [review]

**Review by himanshu-kun — APPROVED** [review]
/lgtm

**Pull request merged**